src/configs/shared.ts

```diff
@@ -25,3 +25,7 @@ export const DEFAULT_CHAIN_ID = 1;
 export const DEFAULT_RPC_ENDPOINT = "https://rpc-pay.ubq.fi/v1/mainnet";
 export const DEFAULT_PAYMENT_TOKEN = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
 export const DEFAULT_PERMIT_BASE_URL = "https://pay.ubq.fi";
+
+export const RESERVED_USERNAMES: { [index: string]: boolean } = {
+  "ubiquity-bounties": true,
```

Comment:
Seems like an unusual implementation. I would probably do an array of strings.

Reply:
Array is slower for this case, since key:value pairs are O(1), It can be a Set() instead